assessments/CLI_CIRCLE/earthmover.yml

```diff
@@ -0,0 +1,680 @@
+# See accompanying README.md for details about this file.
+version: 2
+
+config:
+  log_level: DEBUG
+  output_dir:  ${OUTPUT_DIR} # this will be a parameter that is passed through from the command line, leave this as-is.
+  memory_limit: 1GB
+  state_file: ./runs.csv
+  show_graph: False
+  parameter_defaults:
+    STUDENT_ID_JOIN_COLUMN: student_engage_id
```

Comment:
i'm not sure if `student_engage_id` is right. I think this is only used if we have a STUDENT_ID_XWALK step, which we don't, so maybe we can remove `parameter_defaults` entirely for now?

Reply:
No objections here!